tests/unit/CoopCorrection.spec.ts

```diff
@@ -202,39 +202,6 @@ describe('Coop Correction component', () => {
     ])
 
     LegalServices.fetchParties = vi.fn().mockResolvedValue([
-      {
-        officer: {
-          id: 1,
-          firstName: 'Joe',
-          lastName: 'Swanson',
-          middleName: 'P',
-          organizationName: '',
-          partyType: 'person',
-          email: 'completing-party@example.com'
-        },
-        deliveryAddress: {
-          'addressCity': 'Scarborough',
-          'addressCountry': 'CA',
-          'addressRegion': 'BC',
-          'addressType': 'mailing',
-          'deliveryInstructions': null,
-          'id': 2660328,
-          'postalCode': 'M1B 4B9',
-          'streetAddress': '34-70 Alford Cres',
-          'streetAddressAdditional': ''
-        },
-        mailingAddress: {
-          'addressCity': 'Scarborough',
-          'addressCountry': 'CA',
-          'addressRegion': 'BC',
-          'addressType': 'mailing',
-          'deliveryInstructions': null,
-          'id': 2660328,
-          'postalCode': 'M1B 4B9',
-          'streetAddress': '34-70 Alford Cres',
-          'streetAddressAdditional': ''
-        },
-        roles: [{ appointmentDate: '2022-04-01', roleType: 'Incorporator' }] },
```

Comment:
Removing this from the parties response is legitimate because the Incorporator is not returned for Coops. This was throwing off a unit test due to the new filtering.
Eg,

https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v2/businesses/CP1002796/parties
![image](https://github.com/user-attachments/assets/9e73009f-16b0-4c38-a11f-3795b605d8ad)